[2022-04-21 14:56:41,901] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: parallel_dag.processing_tasks.task_2 scheduled__2022-04-20T00:00:00+00:00 [queued]>
[2022-04-21 14:56:42,165] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: parallel_dag.processing_tasks.task_2 scheduled__2022-04-20T00:00:00+00:00 [queued]>
[2022-04-21 14:56:42,166] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-21 14:56:42,166] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-21 14:56:42,166] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-21 14:56:42,419] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): task_2> on 2022-04-20 00:00:00+00:00
[2022-04-21 14:56:42,526] {standard_task_runner.py:52} INFO - Started process 128089 to run task
[2022-04-21 14:56:42,823] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag.processing_tasks', 'task_2', 'scheduled__2022-04-20T00:00:00+00:00', '--job-id', '192', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpzv72mc5f', '--error-file', '/tmp/tmp66zoql32']
[2022-04-21 14:56:42,921] {standard_task_runner.py:80} INFO - Job 192: Subtask task_2
[2022-04-21 14:56:45,348] {logging_mixin.py:109} INFO - Running <TaskInstance: parallel_dag.processing_tasks.task_2 scheduled__2022-04-20T00:00:00+00:00 [running]> on host airflowvm
[2022-04-21 14:56:46,701] {taskinstance.py:1446} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=solfa
AIRFLOW_CTX_DAG_ID=parallel_dag.processing_tasks
AIRFLOW_CTX_TASK_ID=task_2
AIRFLOW_CTX_EXECUTION_DATE=2022-04-20T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-20T00:00:00+00:00
[2022-04-21 14:56:49,710] {python.py:175} INFO - Done. Returned value was: None
[2022-04-21 14:56:50,058] {taskinstance.py:1278} INFO - Marking task as SUCCESS. dag_id=parallel_dag.processing_tasks, task_id=task_2, execution_date=20220420T000000, start_date=20220421T145641, end_date=20220421T145650
[2022-04-21 14:56:50,327] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-21 14:56:50,803] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
